### PR TITLE
Colorful layout linter fix

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #

### CODE REVIEW

File: website/patches/auth-astro+4.1.3.patch
Comment (anna-parker) [inline website/patches/auth-astro+4.1.3.patch:1]: Omg this is amazing!!! Can you show me how you did this later in a call?

Comment (fengelniederhammer) [reply website/patches/auth-astro+4.1.3.patch:1]: It's actually simple:
* edit `website/node_modules/auth-astro/client.ts` manually
* run `npx patch-package auth-astro`

Comment (anna-parker) [reply website/patches/auth-astro+4.1.3.patch:1]: And the manual import part? Did you just migrate to the function and paste it into the patch file or was it more involved?

File: website/src/types/Organism.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const organismConfig = {
         backgroundColor: 'bg-indigoMuted',
         backgroundColorFocus: 'group-hover:bg-indigo',
         menuListEntryDecoration: 'hover:decoration-indigo',
-        borderEntryDecoration: 'hover:border-indigo',
+        borderEntryDecoration: 'hover:order-indigo',
```
Comment (anna-parker) [inline website/src/types/Organism.ts:65]:
```suggestion
        borderEntryDecoration: 'hover:border-indigo',
```

**Pull request merged**